spec/page/get_weekly_metrics_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+RSpec.describe 'Fb::Page#get_weekly_metrics' do
+  let(:user) { Fb::User.new access_token: ENV['FB_TEST_ACCESS_TOKEN'] }
+  let(:page) { user.pages.first  }
+
+  context 'given a page and valid metrics' do
+    let(:metrics) { %i(page_impressions page_impressions_paid) }
+
+    it 'returns a hash of given metrics mapped to their values' do
+      weekly_metrics = page.get_weekly_metrics metrics
+      expect(weekly_metrics).to be_a(Hash)
+      expect(weekly_metrics.keys).to all (be_a Symbol)
+      expect(weekly_metrics.keys.sort).to eq metrics.sort
```

Comment:
@S1v4 Is this really a requirement? What happens if the keys are returned in a different order?
Would it affect the code?